app/assets/javascripts/tictactoe.js

```diff
@@ -1 +1,151 @@
 // Code your JavaScript / jQuery solution here
+
+var turn = 0;
+var currentGame = 0;
```

Comment:
Prefer to use `let` or `const` over deprecated `var`! This post provides a bit more info: https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75